docs/dietpi-software-desktop.md

```diff
@@ -85,27 +97,38 @@ Popular Gnome2 based desktop environment.
 
 ![DietPi GNUstep](assets/images/dietpi-software-gnustep-desktop.jpg)
 
-## Chromium - Optional: Web browser
+## Desktop utilities
```

Comment:
**Text below is more to sort thoughts for myself, however read and comment if wanted 😉.**

Jep, moving desktop GUI applications into this category was just the same I though 👍. Its a bid strange since on [dietpi-software](https://github.com/MichaIng/DietPi/blob/dev/dietpi/dietpi-software) we have this category, only with QuiteRSS inside, but as part of "Linux/Additional Software" and not DietPi/Optimised Software", so moving those in code  would move them to the other dialog menu as well... Generally this separation as it is used currently only makes half sense IMO. Two goals got mixed:
1. Have software in the "Additional" menu that is just usual/often used but installed via `apt install <package>` only without any "optimisation" our side.
2. Have software in the "Additional" menu which is usually not directly required by end users, but pulled as a dependency, like build tools/libraries, runtime systems and such.

While the second goal IMO is not unreasonable to keep the "optimised" software menu a bid cleaner, it doesn't mean that we do try to find and use best/"optimised" ways to install those the best possible way with leads overhead, current versions (e.g. over APT repo) etc.
The first goal IMO doesn't make much sense. If it is too easy to install those titles (and we clearly never aim to change anything on top of regular package install), it probably makes more sense to remove them completely and instead make it more clear to users that one can use the Debian repo with its tenth of thousands packages regularly. By times I see new users which are not aware of this. To not cut the current possibilities (also regarding automated firstrun installs) we could add (or replace "additional" menu) with an "Install APT packages" entry, which allows to install additional APT packages just in the same run. A new `dietpi.txt` could then allow to pass a list of those to be installed.